src/efi.rs

```diff
@@ -95,17 +95,25 @@ impl Component for EFI {
     }
 
     fn install(&self, src_root: &str, dest_root: &str) -> Result<InstalledContent> {
-        let meta = if let Some(meta) = get_component_update(src_root, self)? {
+        let meta = if let Some(meta) = get_component_update(src_root, self).with_context(|| {
+            format!(
+                "getting update metadata from {} for {}",
+                src_root,
+                self.name()
+            )
+        })? {
```

Comment:
I think I prefer the patch from https://github.com/coreos/bootupd/pull/89 which would cover this path (there's not much that can fail other than parsing), but also OK to keep it.

Reply:
Sure, dropped!